sdk/batch/azure-mgmt-batch/azure/mgmt/batch/operations/_batch_account_operations.py

```diff
@@ -162,19 +162,17 @@ def get_long_running_output(response):
     create.metadata = {'url': '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Batch/batchAccounts/{accountName}'}
 
     def update(
-            self, resource_group_name, account_name, tags=None, auto_storage=None, custom_headers=None, raw=False, **operation_config):
```

Comment:
where is tag and auto_storage?
I didn't see any change in BatchAccountUpdateParameters for tag and auto_storage

Reply:
The auto gen must have not honored           
"x-ms-client-flatten": true,
          "$ref": "#/definitions/BatchAccountProperties",
          "description": "The properties associated with the account."